src/containers/create-post/create-post.component.jsx

```diff
@@ -4,23 +4,50 @@ import { withRouter } from "react-router-dom";
 import Navbar from "containers/navbar";
 import { useSelector } from "react-redux";
 import Sidebar from "containers/sidebar";
+import { storage } from "database";
 
 const CreatePost = ({ history, sidebarToggle, logout }) => {
   const [postText, setPostText] = useState("");
   const [option1, setOption1] = useState("");
   const [option2, setOption2] = useState("");
+  const [imageOption1, setImageOption1] = useState("");
+  const [imageOption2, setImageOption2] = useState("");
+  const [pictureUpload, setPictureUpload] = useState(false);
   const showSidebar = useSelector((state) => state.sidebarView.sidebarView);
   const user = useSelector((state) => state.users.user);
   const userID = user.userId;
   const username = user.username;
-  const avatarLink = user.avatar_link;
 
   const addPostToDB = () => {
-    addPost({ postText, option1, option2, userID, username, avatarLink });
-    setPostText("");
-    setOption1("");
-    setOption2("");
-    history.push("/");
+    if (pictureUpload && (!imageOption1 || !imageOption2)) {
+      alert("Please upload 2 images");
+    } else if (!pictureUpload && !postText) {
+      alert("Please complete post with text");
+    } else if (!option1 || !option2) {
+      alert("Please fill in options");
+    } else {
+      let imgPost = false;
```

Comment:
Not a huge deal but better if you can avoid using let like we discussed earlier.

Reply:
I tried.